notebooks/demo/accelerating_networkx.ipynb

```diff
@@ -0,0 +1,614 @@
+      "execution_count": null,
+      "metadata": {
+        "id": "9CdYNk62E1v_"
+      },
+      "outputs": [],
+      "source": [
+        "nx.config.backend_priority=[]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Jo39YxVmYolq"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "nx_pr_results = nx.pagerank(G)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sV6dM8ToZDiC"
+      },
+      "source": [
+        "We could set `nx.config.backend_priority` again to list `\"cugraph\"` as the backend, but let's instead show how the `backend` kwarg can be used to override the priority list and force a specific backend to be used."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "oMSvQVGKY0rn"
+      },
+      "outputs": [],
+      "source": [
+        "%%time\n",
+        "nxcg_pr_results = nx.pagerank(G, backend=\"cugraph\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZGux_8xFZneI"
+      },
+      "source": [
+        "In this example, instead of plotting the graph to show that the results are identical, we can compare them directly using the saved values from both runs."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "RcmtdFy4Zw7p"
+      },
+      "outputs": [],
+      "source": [
+        "sorted(nx_pr_results) == sorted(nxcg_pr_results)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mefjUEAnZ4pq"
+      },
+      "source": [
+        "# Working with Bigger Data"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yLY-yl6PuNYo"
+      },
+      "source": [
+        "Now we'll look at a larger dataset from https://snap.stanford.edu/data/cit-Patents.html which contains citations across different U.S. patents granted from January 1, 1963 to December 30, 1999. The dataset represents 16.5M citations (edges) between 3.77M patents (nodes).\n",
+        "\n",
+        "This will demonstrate that data of this size starts to push the limits of the default pure-Python NetworkX implementation."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "lyYF0LbtFwjh"
+      },
+      "outputs": [],
+      "source": [
+        "# The locale encoding may have been modified from the plots above, reset here to run shell commands\n",
+        "import locale\n",
+        "locale.getpreferredencoding = lambda: \"UTF-8\"\n",
+        "!wget https://data.rapids.ai/cugraph/datasets/cit-Patents.csv  # Skip if cit-Patents.csv already exists.\n",
+        "# !wget https://snap.stanford.edu/data/cit-Patents.txt.gz  # Skip if cit-Patents.txt.gz already exists."
```

Comment:
Why do we have this line if it's commented out?

Reply:
It was to give the users the option to download the compressed dataset directly from the SNAP source. I believe it was relevant in our recording as an option.